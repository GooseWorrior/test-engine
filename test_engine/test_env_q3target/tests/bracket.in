

a
q
()

a
q
(())

a
q
()()

a
q
(()())

a
q
((())()((())))

a
q
a

b
ab
a
q
(a)
a
b
q
(ab)
ab
a
q
((a))
a
b
q
(a)(b)
ab
b
q
((a)(b))
ab
b
q
(a)b
ab
b
q
a(b)
ab
a
q
((())()(((a))))
a
b
q
