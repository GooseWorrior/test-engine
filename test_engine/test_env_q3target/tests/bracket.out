
true
false

true
false

true
false
Concatenation (,)
true
false
Concatenation (,)
true
false
Concatenation (,Concatenation (,))
true
false
a
false
false
false
true
a
true
false
Concatenation (a,b)
true
false
a
true
false
Concatenation (a,b)
true
false
Concatenation (a,b)
true
false
Concatenation (a,b)
true
false
Concatenation (a,b)
true
false
Concatenation (,Concatenation (,a))
true
false
